bindings/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Introduction
+**matrix-rust-sdk** leverages [UniFFI](https://mozilla.github.io/uniffi-rs/) to generate bindings for host languages (eg. Swift and Kotlin).
+
+The process requires to developers to write a `.udl` (UniFFI Definition Language) file, where Rust definitions to be exposed are collected.
+
+
+Rust code related with bindings live in the [matrix-rust-sdk/bindings](https://github.com/matrix-org/matrix-rust-sdk/tree/main/bindings) folder.
+
+Each Rust crate can expose definitions to UniFFI using two different approaches:
+- Using an `.udl` file. When a crate has one, you find it under the `src` folder (an example is [here](https://github.com/matrix-org/matrix-rust-sdk/blob/main/bindings/matrix-sdk-ffi/src/api.udl)). In this case most of the Rust definitions can be found in the file.
```

Comment:
I think "most of the Rust definitions" is going to change fairly soon, so we shouldn't word it like this. The attributes are preferred, where applicable.